reference/orders.v2.oas2.yml

```diff
@@ -2635,6 +2724,39 @@ components:
               value:
                 - status: 404
                   message: The requested resource was not found.
+    orderConsignmentlineitemsResp:
+      description: The response for orders that contain consignments with line items.
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              type: object
+              properties:
+                pickups:
+                  type: array
+                  description: Pickup consignments.
+                  items:
+                    $ref: '#/components/schemas/pickupConsignment_Get'
+                shipping:
+                  type: object
+                  description: 'ID of the shipping consignment.'
+                  items:
+                    $ref: '#/components/schemas/shippingConsignment_Base'
+                downloads:
+                  type: array
+                  maxItems: 1
+                  description: Digital consignments for products that are downloaded.
+                  items:
+                    $ref: '#/components/schemas/digitalConsignment_Get'
+                email:
+                  type: object
+                  description: Email consignments for gift certificates.
+                  properties:
+                    gift_certificates:
+                      type: array
+                      items:
+                        $ref: '#/components/schemas/giftCertificateConsignment_Get'
```

Comment:
is it possible to extract this and https://github.com/bigcommerce/docs/blob/b31254b9c1a2db844c1274875dd1f8f7f27d0be1/reference/orders.v2.oas2.yml#L5159-L5183 into a common schema?